### ISSUE TITLE
Included service provider certificate expired on December 1, 2023

### ISSUE THREAD

**Issue by ronnieholm** [digst/OIOSAML.Net#77]
The `SP and WSC (Oiosaml-net.dk TEST) (funktionscertifikat)` referenced in `Web.config` expired on December 1, 2023:

      <SigningCertificate findValue="a402bb172929ae0d0ada62f6864329c35dc29483" storeLocation="LocalMachine" storeName="My" x509FindType="FindByThumbprint"/>

The `WebsiteDemo` is no longer runnable and fails with

    Found no valid certificate configured in the certificate configuration. Make sure at least one valid certificate is configured.

**Comment by AnujBanka** [comment]
@ronnieholm are you able to get valid certificates?

If yes, please share the details.

Thanks in advance

**Comment by ronnieholm** [comment]
@AnujBanka No, I don't know how to get a valid certificate. If you're working on an actual integration, I'd reach out to Digitaliseringsstyrelsen NemLogIn support. They can ask their vendor to update the certificate in this repository.

**Comment by AnujBanka** [comment]
Thanks @ronnieholm . I will try but their support is prolonged.

If you know, can I use the OCES certificate in the sample app?
